pkg/kubelet/kubelet_node_status.go

```diff
@@ -179,7 +179,7 @@ func (kl *Kubelet) reconcileExtendedResource(initialNode, node *v1.Node) bool {
 	if kl.containerManager.ShouldResetExtendedResourceCapacity() {
 		for k := range node.Status.Capacity {
 			if v1helper.IsExtendedResourceName(k) {
-				klog.Infof("Zero out resource %s capacity in existing node.", k)
+				klog.InfoS("Zero out resource capacity in existing node", "resourceName", k)
```

Comment:
Should we also include the node name here as a key/value pair?

Reply:
Agree. Log message mentions node, so it's expected that log references node